server/parsec/config.py

```diff
@@ -197,6 +197,9 @@ class BackendConfig:
 
     # Bearer token used to authenticate the administration API
     administration_token: str
+    # Bearer token used to authenticate the service API
+    # If not defined, the API is not accessible
+    service_token: str | None = None
```

Comment:
What is this service API ?

If we end up with two unrelated APIs both using a token, we must further document what each of them are for.